config/config.go

```diff
@@ -167,6 +168,16 @@ func newDefaultMetricsConfig() *MetricsConfig {
 	}
 }
 
+type MessagePoolConfig struct {
+	Limit int `json:"limit"`
```

Comment:
Use a more descriptive name, e.g. "maxMessageCount". There will be other limits in the future.